### DISCUSSION TITLE
Adding the NFC keychain/card

### DISCUSSION THREAD

**Discussion by arnoldtaw** [kormax/apple-home-key-reader#37] [Q&A]
I purchased the kit from Aliexpress and it came with a key tag and key card.
 
When I tried tapping the card, it recognized it but it's not accepted which is understandable. How can I add those cards while using the HomeKey/phone? 

`[2024-07-11 23:47:49,686] [    INFO] service           :126  Found non-ISODEP Tag with UID: XXXXXXXXX
`

**Comment by kormax** [comment]
Custom tags are not supported by HomeKit.

But there's nothing preventing you from modifying the code to trigger "door unlock" when a tag with a known UID is tapped against the lock.
